### ISSUE TITLE
Automate or script gas benchmarks extraction

### ISSUE THREAD

**Issue by gjermundgaraba** [cosmos/solidity-ibc-eureka#103]
It would be good to have a standard "one command" method for getting all the gas benchmarks. I am not sure we all get the benchmarks the same way from the same tests.

That way, we can also have a CI job running and updating the benchmarks for all changes so we can get notifications if something is unexpectedly changing (for good or bad).

What we need to decide on is how we get the gas measurements and from where then we can create a script or something that extracts it to a useful format with the exact numbers we need.

**Comment by srdtrk** [comment]
Yes, I'm concerned that we don't actually get the benchmarks the same way as well. I'd be for a github bot that comments on each PR as I don't actually like github bots making commits on PRs but I'm open to everything.
